admin/requirements-ansible.in

```diff
@@ -1,3 +1,3 @@
 ansible>2.6<2.7
-cryptography>=2.3
+cryptography>=2.5
```

Comment:
New comment here? @conorsch

Reply:
Ultimately, the explanation of _why_ we're bumping the version of cryptography should be in a commit message, but for now, go ahead and add a comment to the `.in` file—we can always pull that into a commit message later, as part of review!

Reply:
resolved in `v3 onions: use cryptography>=2.7`